### ISSUE TITLE
Trying to understand the plots for custom object detection

### ISSUE THREAD

**Issue by MLcodie** [ultralytics/yolov5#1632]
Hi, 
I have created a custom object detection for 15 classes. Ran the training without any pre-trained weights for Yolov5s. Now looking at the plots I see that val Box and val Classification graphs are empty. I have 15 classes for my custom training. Want to make sure this is normal?

![image](https://user-images.githubusercontent.com/75185957/101403687-6f5de980-388a-11eb-801b-f0026a451d9e.png)

**Comment by glenn-jocher** [comment]
@MLcodie missing plotting data is simply due to nan values, which is not normal. I don't know what exactly may be causing this in your dataset, but perhaps you should put breakpoints in the loss function in test.py to trace the origin.

**Comment by MLcodie** [comment]
@glenn-jocher one more question how do I get the plots for validation? I believe the plots above is only for training. 

Thank you!

**Comment by MLcodie** [comment]
I was able to figure out what I did wrong. 

Thank you!

**Comment by glenn-jocher** [comment]
Great! Validation is first 3 columns of bottom row.

**Comment by burglarhobbit** [comment]
@MLcodie did you check the plots of `test_batch0_labels.jpg` and so on, are the bounding boxes showing correctly?
